src/empirical.jl

```diff
@@ -47,7 +53,14 @@ evaluate CDF values on other samples.
 `extrema`, `minimum`, and `maximum` are supported to for obtaining the range over which
 function is inside the interval ``(0,1)``; the function is defined for the whole real line.
 """
-ecdf(X::RealVector{T}) where T<:Real = ECDF(sort(X))
+ecdf(X::RealVector) = ECDF(sort(X), weights(Float64[]))
+
+function ecdf(X::RealVector, w::AbstractVector{<:Real})
```

Comment:
Actually we're planning to use a `weights` keyword argument in the future to pass weights to all functions. Could you change the PR to do that?

Reply:
Changed.